iaen.edu.ec,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
iainbengkulu.ac.id,"HTTPSConnectionPool(host='iainbengkulu.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iainbengkulu.ac.id', port=443): Read timed out. (read timeout=2)"
iam.ma,"HTTPSConnectionPool(host='iam.ma', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.iam.ma', port=443): Max retries exceeded with url: /Pages/index.aspx (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff263a40fa0>, 'Connection to www.iam.ma timed out. (connect timeout=2)'))"
ias.co.il,"HTTPSConnectionPool(host='ias.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ias.co.il', port=80): Read timed out. (read timeout=2)"
ibict.br,"HTTPSConnectionPool(host='ibict.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='ibict.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
icbc.com.ar,"HTTPSConnectionPool(host='icbc.com.ar', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='icbc.com.ar', port=443): Read timed out. (read timeout=2)"
icbf.gov.co,"HTTPSConnectionPool(host='icbf.gov.co', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff2610695e0>, 'Connection to icbf.gov.co timed out. (connect timeout=2)'))","HTTPConnectionPool(host='icbf.gov.co', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff261aecd00>, 'Connection to icbf.gov.co timed out. (connect timeout=2)'))"
icm.edu.pl,"HTTPSConnectionPool(host='icm.edu.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='icm.edu.pl', port=443): Read timed out. (read timeout=2)"
idi.org.il,"HTTPSConnectionPool(host='idi.org.il', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff2642112e0>: Failed to establish a new connection: [Errno 113] No route to host'))","HTTPConnectionPool(host='idi.org.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff263aaa4f0>, 'Connection to idi.org.il timed out. (connect timeout=2)'))"
ieatishootipost.sg,"HTTPConnectionPool(host='ieatishootipost.sg', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ieatishootipost.sg', port=80): Read timed out. (read timeout=2)"
iefp.pt,"HTTPSConnectionPool(host='iefp.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff25eec1fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='iefp.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff260066d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
iett.istanbul,"HTTPSConnectionPool(host='iett.istanbul', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='iett.istanbul', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
iform.dk,"HTTPSConnectionPool(host='iform.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iform.dk', port=443): Read timed out. (read timeout=2)"
igm.org.il,"HTTPSConnectionPool(host='igm.org.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='igm.org.il', port=80): Read timed out. (read timeout=2)"
iho.hu,"HTTPSConnectionPool(host='iho.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iho.hu', port=443): Read timed out. (read timeout=2)"
iif.hu,"HTTPSConnectionPool(host='iif.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff25f7b1880>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='iif.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff25e4841f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ijf.hr,"HTTPSConnectionPool(host='ijf.hr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff2625880a0>, 'Connection to ijf.hr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ijf.hr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff25e0844f0>, 'Connection to ijf.hr timed out. (connect timeout=2)'))"
ilkha.com,"HTTPSConnectionPool(host='ilkha.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ilkha.com', port=443): Read timed out. (read timeout=2)"
ilmangione.it,"HTTPSConnectionPool(host='ilmangione.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.ilmangione.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ilmusosial.id,"HTTPSConnectionPool(host='ilmusosial.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff263737250>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ilmusosial.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff25eaf4370>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
imamu.edu.sa,"HTTPSConnectionPool(host='imamu.edu.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='imamu.edu.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
imaret.com.tr,"HTTPSConnectionPool(host='www.imaret.com.tr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='imaret.com.tr', port=443): Read timed out. (read timeout=2)"
imf.org,"HTTPSConnectionPool(host='www.imf.org', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.imf.org', port=443): Read timed out. (read timeout=2)"
imge.com.tr,"HTTPSConnectionPool(host='www.imge.com.tr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.imge.com.tr', port=443): Read timed out. (read timeout=2)"
imged.pl,"HTTPSConnectionPool(host='imged.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff263609c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='imged.pl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff262d35ee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
imist.ma,"HTTPSConnectionPool(host='imist.ma', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff261fd22e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='imist.ma', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff25f4e8c10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
immoproxio.be,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
imprentanacional.go.cr,"HTTPSConnectionPool(host='imprentanacional.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff2637f2fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='imprentanacional.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff261ca9d60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
impreseita.com,"HTTPSConnectionPool(host='impreseita.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff2625fd9d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='impreseita.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff2620e83a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
imt-ip.pt,"HTTPSConnectionPool(host='imt-ip.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff25aa08f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='imt-ip.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff262fb8610>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
imtranslator.net,"HTTPSConnectionPool(host='imtranslator.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='imtranslator.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
imunita.online,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
inafed.gob.mx,"HTTPSConnectionPool(host='inafed.gob.mx', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff2627971c0>, 'Connection to inafed.gob.mx timed out. (connect timeout=2)'))","HTTPConnectionPool(host='inafed.gob.mx', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff263f15340>, 'Connection to inafed.gob.mx timed out. (connect timeout=2)'))"
inclusion.gob.ec,"HTTPSConnectionPool(host='inclusion.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'inclusion.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
indap.gob.cl,"('Connection aborted.', BadStatusLine('\n'))","HTTPConnectionPool(host='www.indap.gob.cl', port=80): Read timed out. (read timeout=2)"
indexlivingmall.com,"HTTPSConnectionPool(host='indexlivingmall.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff25a25eb20>, 'Connection to indexlivingmall.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='indexlivingmall.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff25a390400>, 'Connection to indexlivingmall.com timed out. (connect timeout=2)'))"
indiaglitz.com,"HTTPSConnectionPool(host='indiaglitz.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff2635c1190>, 'Connection to indiaglitz.com timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='www.indiaglitz.com', port=443): Read timed out. (read timeout=2)"
indonesia.go.id,"HTTPSConnectionPool(host='indonesia.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.indonesia.go.id', port=443): Read timed out. (read timeout=2)"
ine.gub.uy,"HTTPSConnectionPool(host='ine.gub.uy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='ine.gub.uy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
inefop.org.uy,"HTTPSConnectionPool(host='inefop.org.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff262e16a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='inefop.org.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff262f8c100>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
inews.bg,"HTTPSConnectionPool(host='inews.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='inews.bg', port=443): Read timed out. (read timeout=2)"
infarmed.pt,"HTTPSConnectionPool(host='infarmed.pt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff2625fd100>, 'Connection to infarmed.pt timed out. (connect timeout=2)'))","HTTPConnectionPool(host='infarmed.pt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff2611c2310>, 'Connection to infarmed.pt timed out. (connect timeout=2)'))"
infinitydesign.in.th,"HTTPSConnectionPool(host='infinitydesign.in.th', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.infinitydesign.in.th', port=443): Read timed out. (read timeout=2)"
infocenters.co.il,"HTTPSConnectionPool(host='infocenters.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infocenters.co.il', port=443): Read timed out. (read timeout=2)"
infogate.cl,"HTTPSConnectionPool(host='infogate.cl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infogate.cl', port=443): Read timed out. (read timeout=2)"
infoguia.co.cr,"HTTPSConnectionPool(host='infoguia.co.cr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff2611abca0>, 'Connection to infoguia.co.cr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='infoguia.co.cr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff262782820>, 'Connection to infoguia.co.cr timed out. (connect timeout=2)'))"
infoguia.com,"HTTPSConnectionPool(host='infoguia.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff26326c730>, 'Connection to infoguia.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='infoguia.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff260fc4cd0>, 'Connection to infoguia.com timed out. (connect timeout=2)'))"
infojuht.ee,"HTTPSConnectionPool(host='infojuht.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='infojuht.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
infomed.co.il,"HTTPSConnectionPool(host='infomed.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff263375160>, 'Connection to infomed.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='infomed.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff261630cd0>, 'Connection to infomed.co.il timed out. (connect timeout=2)'))"
infonegocios.biz,"HTTPSConnectionPool(host='infonegocios.biz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infonegocios.biz', port=443): Read timed out. (read timeout=2)"
infonegocios.info,"HTTPSConnectionPool(host='infonegocios.info', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infonegocios.info', port=443): Read timed out. (read timeout=2)"
infoprision.com,"HTTPSConnectionPool(host='infoprision.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'infoprision.com' doesn't match 'www.infoprision.com'"")))","HTTPSConnectionPool(host='infoprision.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'infoprision.com' doesn't match 'www.infoprision.com'"")))"
infoproff.com,"HTTPSConnectionPool(host='infoproff.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.infoproff.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
infopublik.id,"HTTPSConnectionPool(host='infopublik.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infopublik.id', port=443): Read timed out. (read timeout=2)"
informa.es,"HTTPSConnectionPool(host='www.informa.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.informa.es', port=443): Read timed out. (read timeout=2)"
ing.be,"HTTPSConnectionPool(host='ing.be', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ing.be', port=80): Read timed out. (read timeout=2)"
injournal.rs,"HTTPSConnectionPool(host='injournal.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='injournal.rs', port=80): Read timed out. (read timeout=2)"
ins-cr.com,"HTTPSConnectionPool(host='ins-cr.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff261ca95e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ins-cr.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff25a629b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
inta.gob.ar,"HTTPSConnectionPool(host='www.argentina.gob.ar', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='inta.gob.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff262323040>: Failed to establish a new connection: [Errno 111] Connection refused'))"
interbuh.com.ua,"HTTPSConnectionPool(host='interbuh.com.ua', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='interbuh.com.ua', port=443): Read timed out. (read timeout=2)"
intercars.com.pl,"HTTPSConnectionPool(host='intercars.com.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='intercars.com.pl', port=443): Read timed out. (read timeout=2)"
intermarche.pt,"HTTPSConnectionPool(host='www.intermarche.pt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.intermarche.pt', port=443): Read timed out. (read timeout=2)"
intima.hu,Exceeded 30 redirects.,Exceeded 30 redirects.
investireoggi.it,"HTTPSConnectionPool(host='www.investireoggi.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.investireoggi.it', port=443): Read timed out. (read timeout=2)"
invima.gov.co,"HTTPSConnectionPool(host='invima.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='invima.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
inzercia.sk,"HTTPSConnectionPool(host='www.inzercia.sk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.inzercia.sk', port=443): Read timed out. (read timeout=2)"
io.ua,"HTTPSConnectionPool(host='io.ua', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff25ad56b50>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='io.ua', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff26265f2e0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
ioc.ee,"HTTPSConnectionPool(host='ioc.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.ioc.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ipapercms.dk,"HTTPSConnectionPool(host='ipapercms.dk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff25d8a4340>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ipapercms.dk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff25a0b8940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ipleaders.in,"HTTPSConnectionPool(host='ipleaders.in', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ipleaders.in', port=443): Read timed out. (read timeout=2)"
ipleiria.pt,"HTTPSConnectionPool(host='ipleiria.pt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='ipleiria.pt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ipoor.org,"HTTPSConnectionPool(host='ipoor.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff2616a0c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ipoor.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff25a4133a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ipsantarem.pt,"HTTPSConnectionPool(host='ipsantarem.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff25a8ab340>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ipsantarem.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff25f1647c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ipst.ac.th,"HTTPSConnectionPool(host='ipst.ac.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff259f6f2e0>, 'Connection to ipst.ac.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ipst.ac.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff264235430>, 'Connection to ipst.ac.th timed out. (connect timeout=2)'))"
ipvc.pt,"HTTPSConnectionPool(host='ipvc.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff261229640>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ipvc.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff259ffa4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ir.lv,"HTTPSConnectionPool(host='ir.lv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ir.lv', port=443): Read timed out. (read timeout=2)"
ird.gov.lk,"HTTPSConnectionPool(host='ird.gov.lk', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff25688df70>, 'Connection to ird.gov.lk timed out. (connect timeout=2)'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
